knative-operator/deploy/resources/knativekafka/kafkachannel-v0.17.1.yaml

```diff
@@ -452,7 +452,7 @@ spec:
       serviceAccountName: kafka-ch-controller
       containers:
       - name: controller
-        image: gcr.io/knative-releases/knative.dev/eventing-contrib/kafka/channel/cmd/channel_controller@sha256:5fe9375f42e6353582acb5b70ae4470aef78f55d482d7809714085fc2e5b9782
+        image: registry.svc.ci.openshift.org/openshift/knative-v0.17.1:knative-eventing-sources-kafka-channel-controller
```

Comment:
let's revert this - we have now the CSV override in place

Reply:
@matzew does this mean all image's back to gcr.io's? or just the two you listed?

Reply:
All images I believe.